inrupt/pod-browser
# Feat: accept consent request

# Accept consent request

In this PR: 
- add API call to accept a consent request
- fix mixed up tests in Profile and Avatar components

### To test:

_Note_: you still can only perform requests to **your own Pod**

1 - Generate a consent request (this will work for one or multiple resources, try both just to be safe)
2 - Grab the consent request URL and navigate back to Privacy, put the URL in the field and click "View test request"
3 - Select one or multiple resources (again, try different options and combinations to be safe: request two and accept one, request two and accept two, request one and accept one) and click the confirm button.
4 - You should be redirected to the privacy page with a query param `signedVcUrl` in the URL that matches the signed VC id.


# Checklist
